front/public/locales/en/stdcm.json

```
@@ -91,7 +91,8 @@
     "noResults": "No path found",
     "noScheduledPoint": "The calculation requires either the origin or destination schedule.",
     "pathfindingFailed": "No path have been found for these waypoints.",
-    "requestFailed": "Calculation error for last minute train path"
+    "requestFailed": "Calculation error for last minute train path",
+    "sameOriginAndDestination": "The origin and destination cannot be the same."
```

nitpick: avoiding negation makes sometimes a clearer sentence?

```suggestion
    "sameOriginAndDestination": "The origin and destination must be different."
```

---

Fixed in https://github.com/OpenRailAssociation/osrd/pull/9931/commits/9719af2565978331c107cd86b909b6b3c5aab73b